### ISSUE TITLE
Keywords autocomplete is not working?

### ISSUE THREAD

**Issue by genkimaps** [genkimaps/gis-hub#117]
Seems to be out of order now.  No errors in browser console.

**Comment by cole-fields** [comment]
I haven't noticed this error and was uploading a bunch of PDF resources as documents today.

**Comment by genkimaps** [comment]
@cole-fields 
You're right, false alarm. Don't know what I was thinking. Will delete this.
